src/plasma-cash/user.ts

```diff
@@ -112,6 +168,67 @@ export class User extends Entity {
     return this.getCurrentBlockAsync()
   }
 
+  // Whenever a new block gets submitted refresh the user's state for their coins
+  async watchBlocks() {
```

Comment:
`watchBlocks` itself doesn't need to be `async`

Reply:
I'm making some awaits inside, that's why